Inorder.java

```diff
@@ -0,0 +1,48 @@
+
+/*
+Time Complexity :  O(n^2) where finding the index is O(n) and buildig the arrays is O(n).
+Space Complexity: O(1) as we are not using any extra auxillary space.
```

Comment:
Please try to optimize on time. Please check the space complexity.

Reply:
I am confused what is space complexity in this. Could you please guide me?
As per me it is O(n).

Reply:
You are using  `Arrays.copyOfRange()` here. So this will need additional space.

Reply:
Okay, got it.
Thank You.